packages/react-components/src/components/Modal/ModalFooter.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+interface ModalFooterProps {
+  children: React.ReactNode;
+  className?: string;
+}
+
+export const ModalFooter: React.FC<ModalFooterProps> = ({
```

Comment:
I'm not quite sure what the purpose of this component is. At the moment, it doesn't seem to offer any benefits to the developer. Are there any plans to include additional styling or features in the future?

Reply:
I know that creating separate components for the header and footer was my suggestion, but after thinking about this, in the current state, that footer component doesn't bring any benefit, so I believe we can drop that pattern.
Maybe for the moment it will be enough if you can pass the class to the footer to be able to style it?
WDYT? @VadymBezpalko

Reply:
@marcinsawicki ok 👌